deps/libcypher-parser/lib/src/parser.leg

```diff
@@ -343,7 +343,9 @@ call-clause =
           )*
         | '-'
         ) -
-      )? >                             { $$ = call_clause(p); }
+      )?
+      (WHERE w:expression | w:_null_) >
```

Comment:
Simple enough, it is nice we're getting some experience with the actual parser grammar definition